Source/Scene/Model.js

```diff
@@ -322,8 +324,16 @@ define([
 
                 if (gltf instanceof Uint8Array) {
                     // Binary glTF
+                    var result = parseBinaryGltfHeader(gltf);
```

Comment:
When CESIUM_binary_glTF is first encountered, add a deprecation warning (see the [guide](https://github.com/AnalyticalGraphicsInc/cesium/wiki/Deprecation-Guide)) that support will be removed in Cesium 1.18, and users should reconvert their models using KHR_binary_glTF, which is the new default option for the Cesium model converter: http://cesiumjs.org/convertmodel.html

CC #3102